docs/hacking.md

```diff
@@ -0,0 +1,240 @@
+Hacking the Aeternity Codebase
+
+- Building
+  - See [Build](build.md)
+    - Ubuntu dependencies: `sudo apt install autoconf build-essential cmake
+      erlang libsodium-dev libgmp-dev`
+    - Mac OS: `brew install erlang@24 openssl libsodium autoconf gmp cmake
```

Comment:
Picking the _oldest_ OTP-version we support is a sure way to make it outdated as quickly as possible 😅